fern/assets/dashboard-referral.js

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable no-shadow */
+!(function () {
+  "use strict";
+  const e = {
+    utm_source: "docs",
+    utm_medium: "referral",
+    utm_campaign: "docs_to_dashboard",
+  };
+  function t() {
+    const e = window.location.href,
+      t = window.location.pathname,
+      n = e.match(/\/docs\/(.+)$/);
+    return n
+      ? n[1].replace(/\//g, "_")
+      : t.replace(/^\//, "").replace(/\//g, "_") || "homepage";
+  }
+  function n(t, n) {
+    try {
+      const o = new URL(t);
+      return (
+        o.searchParams.set("utm_source", e.utm_source),
+        o.searchParams.set("utm_medium", e.utm_medium),
+        o.searchParams.set("utm_campaign", e.utm_campaign),
+        o.searchParams.set("utm_content", n),
+        o.toString()
+      );
+    } catch (e) {
+      return console.warn("Error adding UTM parameters to URL:", t, e), t;
```

Comment:
This will output in the user's browser console which seems spammy and unhelpful. Ideally we setup Sentry for Docs or something but can do that another time.

Reply:
Yeah @dslovinsky, best we can do for now! So at least we see the errors in their console and can debug.